src/application-base/private/translationsLoaders.js

```diff
@@ -39,22 +39,26 @@ const loadTranslationsFile = (locale) => {
   }
 };
 
-const loadTranslations = (locale) => {
-  const fallbackLocale = locale.split('-').length > 1 ? locale.split('-')[0] : false;
-
-  return loadTranslationsFile(locale).catch((error) => {
-    logger.warn(`${error.message} Using ${fallbackLocale} data as the fallback locale.`);
+const getFallbackLocale = (locale) => {
+  // there is no fallback for en.
+  if (locale === 'en') {
+    return false;
```

Comment:
Should this return null or undefined instead of false to avoid a mixed return type?

Reply:
good idea

Reply:
```suggestion
    return undefined;
```